region/crateria/east.json

```diff
@@ -347,6 +387,25 @@
                 }
               ]
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get Hit by Thawing Sciser",
+                  "notable": false,
+                  "requires": [
+                    "canTrickyUseFrozenEnemies",
+                    "canDownwardGModeSetup",
+                    {"ammo": {"type": "Super", "count": 1}}
+                  ],
+                  "note": [
+                    "Run from the ledge into the open doorway while hitting the frozen Crab as it thaws and the door transition simultaneously.",
```

Comment:
More precisely, if I understand it right: You hit the frozen crab, then the transition, and the crab unfreezes during the transition.